var/spack/repos/builtin/packages/rose/package.py

```diff
@@ -85,17 +93,46 @@ def languages(self):
 
     def configure_args(self):
         spec = self.spec
-        cc = self.compiler.cc
-        cxx = self.compiler.cxx
+
+        if '+intel_backend' in spec:
+            cc = which('mpicc')
+            cxx = which('mpic++')
```

Comment:
What's the right way to specify the license file in LC?

```
==> Installing intel
==> Using cached archive: /collab/usr/global/tools/rose/release/.spack/var/spack/cache/intel/intel-16.0.3.tgz
==> Already staged intel-16.0.3-c5v6ch6v5hetjibqtxlq2gqrzcmh3wup in /collab/usr/global/tools/rose/release/.spack/var/spack/stage/intel-16.0.3-c5v6ch6v5hetjibqtxlq2gqrzcmh3wup
==> Already patched intel
==> Building intel [Package]
==> Found already existing license /collab/usr/global/tools/rose/release/.spack/etc/spack/licenses/intel/license.lic
==> './install.sh' '--silent' 'silent.cfg'
grep: write error: Broken pipe
The license file(s) you provided is not valid for this product.
--------------------------------------------------------------------------------
==> Executing phase : 'install'
==> Error: ProcessError: Command exited with status 2:
    './install.sh' '--silent' 'silent.cfg'
```

I'm just using the default Intel `16.0.3`:

```
$ which mpicc
/usr/tce/packages/mvapich2/mvapich2-2.2-intel-16.0.3/bin/mpicc
```